examples/MQ/pixelDetector/src/PixelDigiBinSource.cxx

```diff
@@ -38,6 +38,11 @@ PixelDigiBinSource::PixelDigiBinSource(TString inputFileName)
     LOG(debug) << "PixelDigiBinSource created------------";
 }
 
+PixelDigiBinSource::~PixelDigiBinSource()
+{
+    fInputFile.close();
+}
+
```

Comment:
`fInputFile` is a std::ifstream. So its dtor should handle that for us. :-)

Reply:
ok